packages/use-contractkit/src/constants.tsx

```diff
@@ -53,6 +55,16 @@ export const PROVIDERS: {
     showInList: () => true,
     listPriority: () => 0,
   },
+  [SupportedProviders.CeloDance]: {
+    name: 'CeloDance',
+    description:
+      'Send、Vote and Earn rewards within one wallet',
```

Comment:
I think you've used the wrong character here for the `,` after 'Send'.
Also please change the casing to 'Send, vote, and earn'